cypress/integration/mocked/welcome.spec.ts

```diff
@@ -63,7 +63,6 @@ describe('Welcome and set password page', () => {
     );
     cy.mockAll(200, allConsents, CONSENTS_ENDPOINT);
     cy.mockAll(200, verifiedUserWithNoConsent, USER_ENDPOINT);
-    setAuthCookies();
```

Comment:
Previously the backend was not setting cookies on the correct domain for localhost, so these were set again manually in the tests. Now they rely on the cookies set in the response headers